setup.py

```diff
@@ -50,11 +20,11 @@ def run_tests(self):
 DEPENDENCIES = [
     "matplotlib>=3.2.0",
     "Pillow>=6.2.2",  # not our dependency but older versions crash with mpl
-    "numpy>=1.18.0",
-    "pandas==0.25.*",
-    "scipy>=1.4.0",
+    "numpy>=1.19.0",
+    "pandas>=1.0",
+    "scipy>=1.5.0",
     "scikit-learn>=0.22.0",
```

Comment:
scikit-learn>=0.23.0

Reply:
more information plz? i hope you didnt type it on your smartphone so a bit more writing can be expected ...

Reply:
first version compatible with python 3.8